### PR TITLE
Revise credentials

### PR DESCRIPTION
**Pull request by abizjak**
## Purpose

Revise credentials

## Changes

- The issuer now additionally signs the contract address.
- The JSON format of Web3IdCredential now follows W3C "standard".
- There are no more attribute tags (u8). String names are used in proofs and inputs now for Web3Id stuff.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

**Review by eb-concordium — APPROVED** [review]

File: rust-src/concordium_base/src/web3id/mod.rs [hunk 1]
```diff
@@ -1059,7 +1059,12 @@ impl CredentialSchema {
     }
 }
```
Comment (annenkov) [inline rust-src/concordium_base/src/web3id/mod.rs:1060]: Seems like `CredentialSchema` is not used anywhere now, at least not in this file. Maybe it's imported somewhere else, but `attribute_tag` seems irrelevant since tags are not indices now.

**Review by annenkov — APPROVED** [review]
Looks good.

File: rust-src/concordium_base/src/web3id/mod.rs (hunk 1 above)
Comment (abizjak) [reply rust-src/concordium_base/src/web3id/mod.rs:1060]: Forgot to delete this. Will remove.
